### ISSUE TITLE
Item Type CIT aren't working in 1.19.2-Forge-43.2.0

### ISSUE THREAD

**Issue by ZLGaming** [sp614x/optifine#7206]
## Description of Issue
Item type in CIT doesn't work on Forge 43.2.0

## Steps to Reproduce
1. Play Forge 43.2.0
2. Use a texture pack with CIT
3. Attempt to change one of the item's texture

## OptiFine Version
OptiFine_1.19.2_HD_U_H9

## Installation Method
I dropped the JAR file into my mods folder

## Fabric/Forge Version
Forge Version

## Other Installed Mods
https://pastebin.com/bQPYD0yn

## Log Files/Crash Reports
[debug.log](https://github.com/sp614x/optifine/files/10306590/debug.log)

## F3 Debug Screenshot
![2022-12-27_15 20 23](https://user-images.githubusercontent.com/65998833/209627887-8a168be3-db1c-4a9a-abd1-2c37da574052.png)

## Prior Testing
If you've installed using Forge/Fabric:
- Does this happen without other mods installed? 
   - Only OptiFine and Forge

- Does this happen without any mods at all?
   - Only Forge/Fabric by itself.

- Does this happen in OptiFine standalone?
   - No, CIT feature are fine and working in OptiFine version

- Does this happen in Vanilla without OptiFine?
   - N/A

- Have you tried using a binary search to find which mods cause this issue?
   - Yes, until only OptiFine is remaining.
   - Does the issue still happen with only the remaining mods installed?
      - N/A

## Additional Information
Yes, I have checked if: my Custom Items are on, my texture pack is at most top and active, and my item type format (and path) are correct. The Armor type in CIT works, just not in Item type.

**Comment by glea94** [comment]
Optifine isn't made for Forge >43.1.52. This report isn't viable for a Forge 43.2.0.
And OptiFine_1.19.2_HD_U_H9 is for Forge 43.1.1 not 43.2.0 You have to stuck on Forge 43.1.52 max if you want your CIT working.

**Comment by omerien** [comment]
TL;DR : You need Forge 43.1.52 to use the latest version of OptiFine (HD U I1 pre4)
